.github/actions/frontend/install/action.yml

```diff
@@ -0,0 +1,20 @@
+name: "Node & Yarn Install"
+description: "Sets up Node.js and runs install"
```

Comment:
is this file being used?

Reply:
No, looks like I accidentally included it in the PR. I've removed it.